GoogleForCreators/web-stories-wp
# Added the pageTemplate to the libraryLayout

## Context


Add focus to the selected pane when 'Browse Page Templates' is selected.

## Summary


Adds the page template reference information to the library layout to determine which element should get focus. 

## Relevant Technical Choices


`libraryLayout.js` only uses `MEDIA, MEDIA3P, TEXT` reference information currently, we need to also use the `PAGE_TEMPLATES` reference information to properly update the tab reference and add focus.

## To-do



## User-facing changes


Before/After:
https://user-images.githubusercontent.com/14057928/165990000-37949091-764f-4c02-8de9-c68bbf4fffcd.mp4

## Testing Instructions






This PR can be tested by following these steps:

1. Go to Dashboard
2. Create a new Story
3. Right click the 'Canvas'
4. Click 'Browse Page Templates'
- Notice design panel changes to the 'Page Templates' tab

On Staging:
- Notice the 'Page Templates' tab is 'highlighted' (blue bar under tab) but the arrow keys and TAB keys do nothing

On Branch:
- Notice the 'Page Templates' tab is 'highlighted' (blue bar under tab) and the design panel / tab is now in focus so the arrow keys and TAB keys now navigate in the design panel 

## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10698
